olive/systems/docker/docker_system.py

```diff
@@ -72,12 +78,18 @@ def __init__(
                         build_context_path = tempdir
                         shutil.copy2(dockerfile_path, build_context_path)
                         shutil.copy2(local_docker_config.requirements_file_path, build_context_path)
-                        self.image = self.docker_client.images.build(
-                            path=build_context_path,
-                            dockerfile=self.BASE_DOCKERFILE,
-                            tag=local_docker_config.image_name,
-                            buildargs=local_docker_config.build_args,
-                        )[0]
+
+                        try:
+                            self.image, build_logs = self.docker_client.images.build(
+                                path=build_context_path,
+                                dockerfile=self.BASE_DOCKERFILE,
+                                tag=local_docker_config.image_name,
+                                buildargs=local_docker_config.build_args,
+                            )
+                            _print_docker_logs(build_logs, logging.INFO)
```

Comment:
Can we print build log with debug level?

Reply:
will do